inductiva/resources/machines.py

```diff
@@ -221,13 +236,28 @@ def __str__(self):
         return f"Elastic Machine Group {self.name} with {self.machine_type} " \
              "machines"
 
-    def start(self):
-        """Starts minimum number of machines."""
+    def start(self,
+              max_idle_time: float = None,
+              auto_terminate: Union[float, str] = None):
+        """Starts minimum number of machines.
+
+        Args:
+            max_idle_time (float): Time in minutes that the machine can remain
+                idle.
+            auto_terminate (float, str): Time to automatically terminate the
+                machines independently of any simulations running there.
+                The time can be a float, indicating the number of hours the
+                machine up until the machine can be up, or an actual timestamp
+                with the format '2024-12-31T00:00:00+00'.
```

Comment:
it is weird to define a str manually in this format.
```suggestion
                with the format '2024-12-31T00:00:00+00'.
```